### ISSUE TITLE
Build failure: floorp-11.6.0

### ISSUE THREAD

**Issue by notohh** [NixOS/nixpkgs#272209]
### Steps To Reproduce

1. `nix flake update`
2. build floorp

### Build log

```
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/iw3g01v29mlr76q6kpsnbw0anwq52rhd-source
source root is source
unpackPhase completed in 1 minutes 0 seconds
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
applying patch /nix/store/9lx1746yy56pn3dpc35l1xr6l3lykbin-env_var_for_system_dir-ff111.patch
patching file toolkit/xre/nsXREDirProvider.cpp
applying patch /nix/store/6wsdh6246xbmlv3jr5q7qpl9ay3acq04-no-buildconfig-ffx121.patch
patching file docshell/base/nsAboutRedirector.cpp
Hunk #1 succeeded at 73 (offset -15 lines).
patching file toolkit/content/jar.mn
Hunk #1 FAILED at 41.
1 out of 1 hunk FAILED -- saving rejects to file toolkit/content/jar.mn.rej
```
### Additional context
happens with both `floorp` and `floorp-unwrapped` on latest nixpkgs

### Notify maintainers
@christoph-heiss 

### Metadata

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.2, NixOS, 24.05 (Uakari), 24.05.20231129.e92039b`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.1`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```
---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by christoph-heiss** [comment]
I’ll look into it, think I know already what went wrong and was probably was bound to happen sooner or later.

Might be fixed with 11.6.1 already (#270455), but that’s stuck in PR limbo since more than a week as usual.

**Comment by wegank** [comment]
Closed by #270455.
